crates/aptos-faucet/core/src/funder/common.rs

```diff
@@ -96,20 +96,25 @@ impl ApiConnectionConfig {
         if let Some(ref key) = self.key {
             Ok(key.private_key())
         } else {
-            aptos_sdk::bcs::from_bytes(&std::fs::read(self.key_file_path.as_path()).with_context(
-                || {
-                    format!(
-                        "Failed to read key file: {}",
-                        self.key_file_path.to_string_lossy()
-                    )
-                },
-            )?)
-            .with_context(|| {
+            let key_bytes = std::fs::read(self.key_file_path.as_path()).with_context(|| {
                 format!(
-                    "Failed to deserialize data in key file: {}",
+                    "Failed to read key file: {}",
                     self.key_file_path.to_string_lossy()
                 )
-            })
+            })?;
+            // decode as bcs first, fall back to a file of hex
```

Comment:
I wrote this code to try both kinds of decoding before I found the code to export the bcs format. This is kinda still useful, should maybe kinda be lifted to common code for reading a key, or should maybe be rolled back as a complication we don't really need.